### DISCUSSION TITLE
High CPU Usage on Website

### DISCUSSION THREAD

**Discussion by yirelav** [vercel/community#3624] [Bug Report]
### Page to Investigate

https://vercel.com/docs/storage/edge-config/edge-config-integrations

### Steps to Reproduce

When accessing https://vercel.com/docs/storage/edge-config/edge-config-integrations of the website, there is a noticeable spike in CPU consumption, leading to high resource utilization. This issue is consistently reproducible when using Google Chrome or Mozilla Firefox on Ubuntu.

Open Google Chrome or Mozilla Firefox
Navigate to https://vercel.com/docs/storage/edge-config/edge-config-integrations
Observe the CPU usage using a system monitoring tool or task manager.

**Comment by BioniCosmos** [comment]
I’m facing the same issue on macOS Chrome. The page takes over 300% of CPU and my device quickly heats up. Restarting the browser or the system cannot solve the issue.

**Comment by BioniCosmos** [comment]
Garbage Collection is triggered constantly.

<img width="1440" alt="performance" src="https://github.com/vercel/community/assets/41363844/ffa20948-f08a-49de-a732-562b8053e948">

**Comment by ghost** [comment]
Temporarily disabling JavaScript after a page has finished loading is a temporary workaround
